misc/v3-task-endpoint-validator-windows/v3-task-endpoint-validator-windows.go

```diff
@@ -36,6 +36,7 @@ type TaskResponse struct {
 	Revision           string              `json:"Revision"`
 	DesiredStatus      string              `json:"DesiredStatus,omitempty"`
 	KnownStatus        string              `json:"KnownStatus"`
+	AvailabilityZone   string              `json:"AvailabilityZone"`
```

Comment:
I believe similar changes are needed in [this](https://github.com/aws/amazon-ecs-agent/blob/master/misc/taskmetadata-validator/taskmetadata-validator.go) file.